src/style.css

```
@@ -0,0 +1,87 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  left: 0;
+  top: 0;
+  display: flex;
+  
+  width: 100%;
+  height: 60px;
+  
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  align-items: center;
+}
+
+.img {
+  height: 40px;
+  width: 40px;
+  padding: 10px 0 10px 50px;
```

add left and right `padding` to `header`

```suggestion
  padding: 10px 0;
```